pkg/security/ebpf/c/exec.h

```diff
@@ -392,8 +395,9 @@ int kprobe_do_exit(struct pt_regs *ctx) {
         struct exit_event_t event = {};
         struct proc_cache_t *cache_entry = fill_process_context(&event.process);
         fill_container_context(cache_entry, &event.container);
-
         send_event(ctx, EVENT_EXIT, event);
+
```

Comment:
I think you're missing the `fill_span_context()` here, and another one is missing in `kprobe_security_bprm_committed_creds` and in `sched_process_fork` but I can't comment there :)